Tutorial.md

```diff
@@ -11,455 +12,240 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
 
-    This resource does not have any attributes. Instead it offers the initial API affordances in the form of the HTTP Link header and haveL links.
+Then we have a description of our API. Where we have mentioned what this is. You can use further headings to break up the description sections.
 
-As with the API Name the resource name can be followed by an arbitrary Markdown formatted description.
+## Resource Groups
 
-#### Action
+Now it's time to start documenting our resources in our API. To start of with, we're going to can use the `Group` keyword inside a heading to specify a group of related resources. This allows us to group all of our resources related to questions in our API together.
 
-    ## Retrieve Entry Point [GET]
+```markdown
+# Group Questions
 
-Here we have just defined an action named "Retrieve Entry Point" utilizing the `GET` [HTTP Request Method][]. As with a resource and API name we can add any arbitrary description here but since the action name is pretty self explanatory we will skip it for now.
+Resource related to questions in the API.
+```
 
-#### Response
+## Resource
 
-    + Response 200
+Inside of questions resource group, we have a resource in our API called "Questions collection". This resource allows you to view a list of questions. We've created a sub-heading inside our group called `Questions collection` which then specifies the URI for this resource inside our API which is placed inside square brackets at the end of the title.
 
-In API Blueprint an action **should** always include at least one response representing the HTTP response message sent back in response to a HTTP request. The response should always bear a [status code][] and possibly an additional [payload][]. Here we have defined the most common response "200" indicating the request has succeeded.
+```markdown
+## Questions collection [/questions]
+```
 
-#### Response Payload
+### Actions
 
-    + Response 200 (application/hal+json)
-        + Headers
+In an API Blueprint, you need to specify the actions you can make on this resource in HTTP. For example, we can retrive the list of questions using the HTTP method `GET`.
 
-                Link: <http:/api.gistfox.com/>;rel="self",<http:/api.gistfox.com/gists>;rel="gists"
+We can create a sub-heading inside our resource with a title for our action, followed by the HTTP method to use inside the square brackets.
 
-        + Body
+```markdown
+### List all questions [GET]
+```
 
-                {
-                    "_links": {
-                        "self": { "href": "/" },
-                        "gists": { "href": "/gists?{since}", "templated": true }
-                    }
-                }
+In API Blueprint, an action **should** always include at least one response representing the HTTP response message sent back in response to a HTTP request. The response should always include a [status code][] and possibly an additional [payload][].
```

Comment:
remove "In API Blueprint", if this is not clear to the readers at this point then they are lost anyway :)